Editor/Scripts/wind_forces_tutorial.py

```diff
@@ -0,0 +1,94 @@
+
+class WindForcesTutorial(Tutorial):
+    def __init__(self):
+        super(WindForcesTutorial, self).__init__()
+
+        self.title = "PhysX Wind Forces"
+
+        self.add_step(TutorialStep("Create Wind Forces", 
+                """<html><p style="font-size:13px">Greetings!<br><br>A <i>wind provider</i> is an entity that 
+                defines a global or localized wind force that can affect certain components such as <b>Cloth</b> components. 
+                <br><br>In this tutorial, we'll set up a local wind provider, add a cloth object to the scene, and configure the cloth object to simulate a sheet gently blowing in the breeze. 
+                Let's begin by creating a new entity for a wind provider. Right click in <b>Entity Outliner</b> and choose <b>Create Entity</b> from the context menu. Name the new entity 
+                <strong><code style="font-size:14px;color:#E44C9A">"Wind Provider"</code></strong>.
+                <br><br>For this tutorial, make sure that you have the <b>Nvidia Cloth Gem</b> and <b>PhysX Gem</b> enabled in your project.</p></html>"""))
+        self.add_step(TutorialStep("Add a Tag component",  """<html><p style="font-size:13px">
+                We need to add a tag to the entity to identify it as a wind provider and specify the type of wind force. We can do this with a Tag component.
+                In Entity Inspector, choose Add Component and select Tag from the components list to add a Tag component to the entity. </p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Create a local wind force tag",  """<html><p style="font-size:13px">A wind provider can create a global wind force for the entire level, or a local wind force that is contained within a volume. Let's create a local wind force.
+                In the Tag component, click the + button to add a new tag element. Enter wind in the new element field to create a local wind force.<br><br> </p></html>""", "InspectorMainWindow"))
+        self.add_step(TutorialStep("Add a tag component for the entity",  """<html><p style="font-size:13px"> Edit the tag value to <b>global_wind</b>.
+                <br><br> There are two types of wind: global and local, used to specify which entities provide wind forces. <br><br>
+                If you choose to use the Local wind tag property, the wind force affects only certain entities.</p></html>""", "EntityOutlinerWidgetUI"))
+        self.add_step(TutorialStep("Add a PhysX collider component",  """<html><p style="font-size:13px"> Wind providers must have a PhysX collider. For local wind forces, the collider specifies the volume that contains the wind force. 
+                Objects are affected by the local wind force only when they are inside the wind provider's PhysX collider.
+                In Entity Inspector, choose Add Component and add a PhysX Collider component to the entity. <br><br> </p></html>""", "InspectorMainWindow"))
```

Comment:
Highlight the Add Component button

```suggestion
        self.add_step(TutorialStep("Add a PhysX Collider component",  """<html>
                <p style="font-size:13px"> Wind providers must have a PhysX 
                collider. For local wind forces, the collider defines the volume
                 that contains the wind force. Objects are affected by the local
                  wind force only when they are inside of the wind provider's 
                PhysX collider.<br><br>In <b>Entity Inspector</b>, choose <b>Add
                 Component</b> and add a <b>PhysX Collider</b> component to the 
                 entity.</p></html>""", "m_addComponentButton"))
```